src/messageDescriptors/search.ts

```diff
@@ -17,7 +17,7 @@ export const search = defineMessages({
     },
     inputLabel: {
         id: 'search.inputLabel',
-        defaultMessage: 'Search newsroom',
+        defaultMessage: 'Search site',
```

Comment:
This one looks weird to me 🤔 
Can you advise with somebody about the copy?

Reply:
I see that we don't even show the label to anyone in themes 🤔 unless in screen readers I believe?

Reply:
Hey @champagne-wizard! Can you advise on this? Maybe just use the word "Search" without site/newsroom appended :)

Reply:
This is for the search bar placeholder text, right? Reckon just "Search" is fine (quick google isn't throwing up much controversy) :)